config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
Which configuration file are we talking about?

> Not sure about using snapd database, it might be an option, which might generic for different bootloaders. In the case of android-boot using it or not depends on whether tools would need to be added to the initramfs or not.

A very simple way to find the current core and kernel snap from the recovery would be by looking at the /snap/core/current and /snap/kernel/current symlink in the writable partition.